src/main/java/io/github/fileanalysissuite/adaptersdk/impls/jaxrs/internal/utils/ItemMetadataFunctions.java

```diff
@@ -60,4 +60,15 @@ public static io.github.fileanalysissuite.adaptersdk.impls.jaxrs.internal.server
 
         return modelItemMetadata;
     }
+
+    private static String toRFC3339DateTimeString(final Instant time)
```

Comment:
On another note I want you to create a new `InstantFunctions.java` module in the `.internal.utils` package, and to make this a `public static` method in that package, rather than making it a `private static` in `ItemMetadataFunctions()`.  It is not really anything to do with `ItemMetadata`.  It is very plausible that another module may require this functionality in the future.